src/app/login/login.component.html

```diff
@@ -1,9 +1,11 @@
 
-Login form <br/>
-Username: <input type="text" name="username" [(ngModel)]="username"/> <br/>
-Password: <input type="password" name="password" [(ngModel)]="password"/> <br/>
+Imię i nazwisko: <input type="text" name="username" [(ngModel)]="username"/> <br/>
+Hasło: <input type="password" name="password" [(ngModel)]="password"/> <br/>
+
+<button (click)="login()">Zaloguj</button> <br/>
```

Comment:
Please use angular material components. It would be nice to use it in whole form, not only the buttons